schemas/deviceToCloud/alert/alert.json

```diff
@@ -0,0 +1,37 @@
+{
+    "title": "Alert",
+    "description": "The device Alert",
+    "type": "object",
+    "properties": {
+      "appId": {
+          "type": "string",
+          "const": "ALERT"
+      },
+      "ts": {
+          "type": "number",
+          "description": "Unix timestamp given in milliseconds when the data was sampled"
+      },
+      "seq": {
+          "type": "number",
+          "description": "Sequence number of the message. This is a counter, used for ordering message if `ts` not available/used"
+      },
+      "desc": {
+          "type": "string",
+          "description": "Description of the alert"
+      },
+      "type": {
+          "type": "number",
+          "format": "int32",
+          "description": "Enmeration for the kind of alert. This value can be user defined. TBD: the range for custom values"
+      },
+      "value": {
+          "type": "number",
+          "format": "float",
+          "description": "This represents the value that triggered the alert. For instance, if a threshold was crossed and this is the violating value"
+      }
+    },
+    "required": [
+       "appId"
```

Comment:
should either "type" or "desc" also be required?
not sure if that is possible to specify one of...

Reply:
Ya I think thats possible. Are you saying that it should either have one of `type` or `desc`?

Reply:
yes, i don't think it makes sense for a device to send an alert without specifying a type or a description of the alert?

Reply:
I agree: `type`

Reply:
Ya that makes sense to me. I will change it

Reply:
Type must be specified -- I do not feel that is optional.

Description is optional to me.

Reply:
i thought "one of" because maybe the device just wants to send some alert text... like "hello world" which doesn't require a `type` value.

Reply:
No, I think it's simpler to always require type.  It is just a number, so it doesn't take much data to send it.  This would make it easier for people processing alerts to filter out such informational alerts.  In that regard, we can make a predefined one to cover this, like "ALERT_TYPE_MSG."

Reply:
ok